src/js/player.js

```diff
@@ -1203,14 +1495,25 @@ class Player extends Component {
       // Don't trigger a change event unless it actually changed
       if (this.controls_ !== bool) {
         this.controls_ = bool;
+
+        if (!this.usingNativeControls()) {
```

Comment:
I think this is backwards. We only want to set the controls on the tech if we *are* using native controls.